src/pages/MetricDetail.svelte

```diff
@@ -90,9 +90,7 @@
 
   <PageTitle text={metric.name} />
 
-  <p>
-    <Markdown text={metric.description} />
-  </p>
+  <Markdown text={metric.description} inline={false} />
```

Comment:
The `text` seems more "HTML"-like if it were in a `slot`, perhaps?

Reply:
Yeah, this is the way we implemented it originally but it needed some hacks to work and had some problems. @Iinh reverted this behaviour in #192

Reply:
(as such I'm going to merge this in, but if you want to follow up on this let me know)